test/fixtures/BD-2015.json

```diff
@@ -0,0 +1,154 @@
+    "name": "স্বাধীনতা দিবস",
+    "type": "public",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2015-04-14 00:00:00",
+    "start": "2015-04-13T18:00:00.000Z",
+    "end": "2015-04-14T18:00:00.000Z",
+    "name": "পহেলা বৈশাখ",
+    "type": "public",
+    "_weekday": "Tue"
+  },
+  {
+    "date": "2015-05-01 00:00:00",
+    "start": "2015-04-30T18:00:00.000Z",
+    "end": "2015-05-01T18:00:00.000Z",
+    "name": "মে দিবস",
+    "type": "public",
+    "_weekday": "Fri"
+  },
+  {
+    "date": "2015-05-19 00:00:00",
+    "start": "2015-05-18T18:00:00.000Z",
+    "end": "2015-05-19T18:00:00.000Z",
+    "name": "বুদ্ধ পূর্ণিমা",
+    "type": "public",
+    "_weekday": "Tue"
+  },
+  {
+    "date": "2015-07-17 00:00:00 -0600",
+    "start": "2015-07-16T12:00:00.000Z",
+    "end": "2015-07-19T12:00:00.000Z",
+    "name": "ঈদুল ফিতর",
+    "type": "public",
+    "_weekday": "Fri"
+  },
+  {
+    "date": "2015-08-15 00:00:00",
+    "start": "2015-08-14T18:00:00.000Z",
+    "end": "2015-08-15T18:00:00.000Z",
+    "name": "জাতীয় শোক দিবস",
+    "type": "public",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2015-08-24 00:00:00",
+    "start": "2015-08-23T18:00:00.000Z",
+    "end": "2015-08-24T18:00:00.000Z",
+    "name": "জন্মাষ্টমী",
+    "type": "public",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2015-09-23 00:00:00 -0600",
+    "start": "2015-09-22T12:00:00.000Z",
+    "end": "2015-09-25T12:00:00.000Z",
+    "name": "ঈদুল আযহা",
+    "type": "public",
+    "_weekday": "Wed"
+  },
+  {
+    "date": "2015-10-08 00:00:00",
+    "start": "2015-10-07T18:00:00.000Z",
+    "end": "2015-10-08T18:00:00.000Z",
+    "name": "দুর্গা পূজা",
+    "type": "public",
+    "_weekday": "Thu"
+  },
+  {
+    "date": "2015-10-23 00:00:00 -0600",
+    "start": "2015-10-22T12:00:00.000Z",
+    "end": "2015-10-23T12:00:00.000Z",
+    "name": "আশুরা",
+    "type": "public",
+    "_weekday": "Fri"
+  },
+  {
+    "date": "2015-11-21 00:00:00",
+    "start": "2015-11-20T18:00:00.000Z",
+    "end": "2015-11-21T18:00:00.000Z",
+    "name": "Armed Forces Day",
+    "type": "observance",
+    "_weekday": "Sat"
+  },
+  {
+    "date": "2015-12-14 00:00:00",
+    "start": "2015-12-13T18:00:00.000Z",
+    "end": "2015-12-14T18:00:00.000Z",
+    "name": "Martyred Intellectuals Day",
+    "type": "observance",
+    "_weekday": "Mon"
+  },
+  {
+    "date": "2015-12-16 00:00:00",
+    "start": "2015-12-15T18:00:00.000Z",
+    "end": "2015-12-16T18:00:00.000Z",
+    "name": "বিজয় দিবস",
+    "type": "public",
+    "_weekday": "Wed"
+  },
+  {
+    "date": "2015-12-23 00:00:00 -0600",
+    "start": "2015-12-22T12:00:00.000Z",
+    "end": "2015-12-23T12:00:00.000Z",
+    "name": "ঈদে মিলাদুন্নবী",
+    "type": "public",
+    "_weekday": "Wed"
+  },
+  {
+    "date": "2015-12-25 00:00:00",
+    "start": "2015-12-24T18:00:00.000Z",
+    "end": "2015-12-25T18:00:00.000Z",
+    "name": "বড়দিন",
+    "type": "public",
+    "_weekday": "Fri"
+  }
+]
```

Comment:
It is probably worth updating the script which generates these files to have a newline at the end of the file to avoid the git warnings.